### PR TITLE
Transform Event "xblock.survey.submitted" add code and JSON

### PR DESCRIPTION
**Pull request by mumarkhan999**
**Trello Link:** [click here](https://trello.com/c/wItWF0S7/44-poll-and-survey-event-xblocksurveysubmitted)

**Description:** Contains code and json files for xblock.survey.submitted event

**Checks before merge:**

- [x] JSON Reviewed
- [x]   Code Reviewed
- [x]   Unit Test Passed
- [x]   Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/transformers/xblock_transformers.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    :return: updated caliper_event.
+    """
+    caliper_object = {
+        "id": current_event['referer'],
+        "type": 'Assessment',
+        "extensions": current_event['event']
+    }
+
+    caliper_event['extensions']['extra_fields'].update({
+        "ip": current_event['ip'],
```
Comment (asamolion) [inline openedx/features/caliper_tracking/transformers/xblock_transformers.py]: Please use single quotes in all of code. Only use double quotes for docstrings.

Comment (mumarkhan999) [reply openedx/features/caliper_tracking/transformers/xblock_transformers.py]: done in the latest commit.

File: openedx/features/caliper_tracking/tests/expected/xblock.survey.submitted.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "object": {
+    "extensions": {
+      "choices": {
+        "enjoy": "Y",
+        "learn": "M",
+        "recommend": "Y"
+      },
+      "url_name": "4b2805d63ca940f98b1311f7f30c316e"
+    },
+    "id": "http://edx.devstack.lms:18000/courses/course-v1:edX+DemoX+Demo_Course/courseware/57cffc9e27e5494aa72444b142cc79fd/da1e0ff2718b471b88ad9a5d5fede8a7/1?activate_block_id=block-v1%3AedX%2BDemoX%2BDemo_Course%2Btype%40vertical%2Bblock%40c1f57e13fe9d4cc7b7780c8fb5d26ff5",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/xblock.survey.submitted.json]: Please replace `edx.devstack.lms` with `localhost`.

Comment (mumarkhan999) [reply openedx/features/caliper_tracking/tests/expected/xblock.survey.submitted.json]: done in the latest commit.

**Pull request merged**